store.html

```diff
@@ -0,0 +1,119 @@
+                    <div class="form-items-holder">
+                 <label for="lastname" class="form-input">Last name *</label>
+                 <input id="lastname" type="text" name="lastname" required maxlength="15">
+                    </div>
+                 </div>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="address" class="form-input">Address *</label>
+                 <input id="address" type="text" name="address" required class="address">   
+                    </div>
+                </div>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="address2" class="form-input">Address 2</label>
+                 <input id="address2" type="text" name="address2" class="address">
+                    </div>
+                </div>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="city" class="form-input">City *</label>
+                 <select id="city" name="city" required>
+                    <option value="-1" selected>Select your city...</option>
+                    <option value="london">London</option>
+                    <option value="milan">Milan</option>
+                    <option value="birmingham">Birmingham</option>
+                    <option value="paris">Paris</option>
+                 </select>
+                    </div>
+                    <div class="store-free-space"></div>
+                    <div class="form-items-holder">
+                <label for="postcode" class="form-input">Postcode *</label>
+                <input id="postcode" type="text" name="postcode" maxlength="8" style="text-transform:uppercase">
+                    </div>
+                </div>
+                <div class="form-line">
+                     <fieldset class="form-input" id="select-colour">
+                        <legend>Select a colour</legend>
+                      <div class="radio-btns">
+                          <div class="radio-lebel-holder">
+                            <input id="orange" type="radio" name="orange" value="orange">
+                            <label for="orange" class="radio-label">Karma orange</label>
+                          </div>
+                          <div class="radio-lebel-holder">
+                           <input id="grey" type="radio" name="orange" value="grey">
+                           <label for="grey" class="radio-label">Space Grey</label>
+                          </div>
+                     </div>
+                    </fieldset>
+                </div>
+                <div class="form-line" id="checkbox-items">
+                 <p class="terms"> <input type="checkbox" class="checkbox-btn" required>By placing your order you agree to Karma's <span style="color:#e0633a">Terms and</span><br><span style="color:#e0633a">Conditions.</span> *</p>
```

Comment:
This is just my recomendation, but I would recommend changing the Terms and condition to a link so when you click on it you would be sent to the non existent Terms and conditions.

Reply:
For sure, next improvement would be this. More time was needed! Thanks for reviewing Kerim